src/tests/ftest/util/test_utils.py

```diff
@@ -202,19 +209,58 @@ def __init__(self, context, log=None, cb_handler=None):
         self.target_list = BasicParameter(None)
         self.scm_size = BasicParameter(None)
         self.nvme_size = BasicParameter(None)
+        # Set USE_API to use API or USE_DMG to use dmg. If it's not set, API is
+        # used.
+        self.control_method = BasicParameter(self.USE_API, self.USE_API)
+        uname = getpass.getuser()
+        gname = grp.getgrnam(uname)[0]
+        self.username = BasicParameter(uname, uname)
+        self.groupname = BasicParameter(gname, gname)
 
         self.pool = None
         self.uuid = None
         self.info = None
         self.svc_ranks = None
         self.connected = False
-
+        # Required to use dmg. It defined the directory where dmg is installed.
+        # Use self.basepath + '/install/bin' in the test
+        self.dmg_bin_path = dmg_bin_path
+        if dmg_bin_path is not None:
+            # We make dmg as the member of this class because the test would
+            # have more flexibility over the usage of the command.
+            self.dmg = DmgCommand(self.dmg_bin_path)
+            self.dmg.insecure.value = True
+            self.dmg.request.value = "pool"
+
```

Comment:
Should we explicitly set `self.dmg = None` if `dmg_bin_path is None`?

Reply:
Yes. I added this variable.